src/wp-content/themes/twentytwentyone/assets/sass/05-blocks/columns/_style.scss

```diff
@@ -6,7 +6,8 @@
 
 	.wp-block-column {
 
-		> * {
+		// Add extra padding, but not for the site title and tagline. 
+		> *:not(.wp-block-site-title):not(.wp-block-site-tagline) {
```

Comment:
I really don't love using these `:not()` rules, but this was the least-complicated way I could think of to ensure these block have the correct spacing. 

Before|After
---|---
<img width="314" alt="Screen Shot 2021-07-01 at 9 08 45 AM" src="https://user-images.githubusercontent.com/1202812/124129627-f56e9700-da4b-11eb-8412-53833bf92e11.png">|<img width="302" alt="Screen Shot 2021-07-01 at 9 07 12 AM" src="https://user-images.githubusercontent.com/1202812/124129624-f4d60080-da4b-11eb-8d15-e5403240fdcc.png">